pkg/processor/sourceprocessor/source_category_filler_test.go

```diff
@@ -66,6 +66,24 @@ func TestFillWithAnnotations(t *testing.T) {
 	assertAttribute(t, attrs, "_sourceCategory", "ABC#123asd#Prefix:sc#from#annot#ns#1#123asd")
 }
 
+// TODO: add more test cases
```

Comment:
Are you planning on doing so in this PR?

Reply:
No, I added tests but I forget to remove this comment :P